requirements.txt

```diff
@@ -1,6 +1,7 @@
+mpi4py
 numpy
+pudb
```

Comment:
I don't think this quite belongs here, since it's not a functional requirement of mirgecom.

Counter-proposal: https://github.com/illinois-ceesd/emirge/pull/59

Reply:
Good point, will remove it here.